libc-test/build.rs

```diff
@@ -468,6 +503,14 @@ fn main() {
             // it's in a header file?
             "endpwent" if android => true,
 
+            // These are either unimplemented or optionally built into uClibc
+            // or "sysinfo", where it's defined but the structs in linux/sysinfo.h and sys/sysinfo.h
+            // clash so it can't be tested, or they're tainted with an old struct definition, as in g"getnameinfo"
+            "mkostemp" | "mkostemps" | "mkstemps" | "futimes" | "setns" | "sysinfo" | "fallocate" | "posix_fallocate" | "preadv" | "pwritev" | "dup3" | "openpty" | "forkpty" |
+            "getloadavg" | "process_vm_readv" | "process_vm_writev" | "backtrace" | "posix_madvise" |
+            "aio_read" | "aio_write" | "aio_fsync" | "aio_error" | "aio_return" | "aio_suspend" | "aio_cancel" | "lio_listio" |
+            "newlocale" | "duplocale" | "freelocale" | "uselocale" | "nl_langinfo_l" | "wcslen" | "getnameinfo" | "wcstombs" if uclibc && linux => true,
```

Comment:
I wonder, should most of these just not be present in the libc crate? Especially the unimplemented ones seem like they shouldn't exist, but including the optional ones is likely ok. It'd be great though if we could verify against a build of uclibc with all optional extensions compiled in.